Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/EfficiencyCondition.mo

```diff
@@ -0,0 +1,298 @@
+    annotation (Line(points={{-58,70},{-42,70}},
+      color={0,0,127}));
+  connect(div.y, hys2.u)
+    annotation (Line(points={{-58,30},{-42,30}},
+      color={0,0,127}));
+  connect(truDel1.u, hys1.y)
+    annotation (Line(points={{-2,70},{-18,70}},
+      color={255,0,255}));
+  connect(truDel.u, hys2.y)
+    annotation (Line(points={{-2,30},{-18,30}},
+      color={255,0,255}));
+  connect(intToRea.u, uTyp)
+    annotation (Line(points={{-82,-100},{-140,-100}},
+      color={255,127,0}));
+  connect(extIndSig.u, intToRea.y)
+    annotation (Line(points={{-42,-100},{-58,-100}},
+      color={0,0,127}));
+  connect(extIndSig.index, uAvaUp)
+    annotation (Line(points={{-30,-112},{-30,-140},{-140,-140}},
+      color={255,127,0}));
+  connect(greThr.u, extIndSig.y)
+    annotation (Line(points={{-2,-100},{-18,-100}},
+      color={0,0,127}));
+  connect(greThr.y, logSwi.u2)
+    annotation (Line(points={{22,-100},{30,-100},{30,50},{38,50}},
+      color={255,0,255}));
+  connect(truDel.y, logSwi.u3)
+    annotation (Line(points={{22,30},{36,30},{36,42},{38,42}},
+      color={255,0,255}));
+  connect(truDel1.y, logSwi.u1)
+    annotation (Line(points={{22,70},{36,70},{36,58},{38,58}},
+      color={255,0,255}));
+  connect(and2.y, yEffCon)
+    annotation (Line(points={{102,0},{140,0}},
+      color={255,0,255}));
+  connect(logSwi.y, and2.u1)
+    annotation (Line(points={{62,50},{70,50},{70,0},{78,0}},
+      color={255,0,255}));
+  connect(logSwi1.y, and2.u2)
+    annotation (Line(points={{62,-30},{70,-30},{70,-8},{78,-8}},
+      color={255,0,255}));
+  connect(con1.y, logSwi1.u1)
+    annotation (Line(points={{22,-10},{36,-10},{36,-22},{38,-22}},
+      color={255,0,255}));
+  connect(hys.y, logSwi1.u3)
+    annotation (Line(points={{22,-50},{36,-50},{36,-38},{38,-38}},
+      color={255,0,255}));
+  connect(logSwi1.u2, greThr.y)
+    annotation (Line(points={{38,-30},{30,-30},{30,-100},{22,-100}},
+      color={255,0,255}));
+  connect(div.u1, uCapReq) annotation (Line(points={{-82,36},{-100,36},{-100,60},
+          {-140,60}}, color={0,0,127}));
+  connect(div1.u1, uCapReq) annotation (Line(points={{-82,76},{-100,76},{-100,60},
+          {-140,60}}, color={0,0,127}));
+
+annotation (
+  defaultComponentName = "effCon",
+  Icon(
+    coordinateSystem(extent={{-100,-100},{100,100}}),
+    graphics={Rectangle(
+                extent={{-100,-100},{100,100}},
+                lineColor={0,0,127},
+                fillColor={255,255,255},
+                fillPattern=FillPattern.Solid),
+              Text(
+                extent={{-120,146},{100,108}},
+                lineColor={0,0,255},
+                textString="%name")}),
+  Diagram(
+    coordinateSystem(
+      preserveAspectRatio=false,
+      extent={{-120,-160},{120,120}})),
+  Documentation(
+    info="<html>
+    <p>
+    Efficiency condition used in staging up and down for boiler plants with both
+    condensing and non-condensing boilers. Implemented according to the
```

Comment:
@milicag : Would this be the correct way to represent the exact points in sequence which define the Efficiency Condition? I didn't want to repeat the main section number too many times

Reply:
That totally works.

Reply:
Sorry the comment got added in the wrong location. The subsection numbers are on the next line, and can't be seen in the image above